tpl/template_funcs.go

```diff
@@ -1699,13 +1699,21 @@ func countRunes(content interface{}) (int, error) {
 }
 
 // humanize returns the humanized form of a single word.
+// If the word given can be identified as an integer, the behavior
+// is to add the corresponding ordinal.
 // Example:  "my-first-post" -> "My first post"
+// Example:  "103" -> "103rd"
 func humanize(in interface{}) (string, error) {
 	word, err := cast.ToStringE(in)
 	if err != nil {
 		return "", err
 	}
-	return inflect.Humanize(word), nil
+
+	_, err = strconv.Atoi(word)
```

Comment:
Note: I used `strconv.Atoi` here instead of `cast.ToIntE` because `ToIntE` degrades a little more gracefully on some conditions than we want in this case. Floats and bools for example.